src/utils/sample/discrete_gauss.rs

```diff
@@ -362,15 +423,20 @@ mod test_sample_d {
     use flint_sys::fmpz_mat::fmpz_mat_hnf;
     use std::str::FromStr;
 
+    use super::sample_d_precomputed_gso;
+
     /// Ensures that the doc-test compiles and runs properly
```

Comment:
Ensure that all comments on the tests start with a capital letter and end with a dot